### ISSUE TITLE
Set Map Focus not present?

### ISSUE THREAD

**Issue by pleonard212** [scholarslab/Neatline#173]
Apologies if I'm missing something obvious, but I was trying to figure out if I could set the zoom/positioning for each record when clicked, as described in README.md:

```
 Set Map Focus: The map focus for a record is the position and zoom that the map viewport is set to when the record is activated by any action elsewhere in the exhibit. For example, if a user clicks on a listing for the record in the description tray or scrolls to the item using the next and previous arrows, the map will automatically refocus at the latitude/longitude and zoom level defined by the map focus for the item.
```

It sounds like it should be in the "Styling" tab, but I'm not seeing it  (both on objects that do and do not have an actual geo item).

Thanks!

**Comment by davidmcclure** [comment]
Yep, the button was accidentally removed in 419b8c1b1289a1c58b99aa0422534677c4095a29. Thanks for catching this - I'll push out a new release today and notify here when it's up.

**Comment by davidmcclure** [comment]
@pleonard212 This is fixed in version 1.1.1 - http://omeka.org/add-ons/plugins/neatline/. Again, thanks for notifying us about this.
